pelias.json

```diff
@@ -78,7 +78,7 @@
       "datapath": "/data/whosonfirst",
       "importVenues": false,
       "importPostalcodes": true,
-      "importPlace": "101715829",
+      "importPlace": "85950361",
```

Comment:
please don't change that here. this file should still be working for portland. we'll start a new directory for additional city config files in the future.